packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -752,9 +752,14 @@ export class ProjectLifecycleManager {
     if (this._projectRoot && testingType && await this.waitForInitializeSuccess()) {
       this.setAndLoadCurrentTestingType(testingType)
 
-      if (testingType === 'e2e' && !this.ctx.migration.needsCypressJsonMigration() && !this.isTestingTypeConfigured(testingType)) {
-        // E2E doesn't have a wizard, so if we have a testing type on load we just create/update their cypress.config.js.
-        await this.ctx.actions.wizard.scaffoldTestingType()
+      if (!this.isTestingTypeConfigured(testingType)) {
+        if (testingType === 'e2e' && !this.ctx.migration.needsCypressJsonMigration()) {
+          // E2E doesn't have a wizard, so if we have a testing type on load we just create/update their cypress.config.js.
+          await this.ctx.actions.wizard.scaffoldTestingType()
+        } else if (testingType === 'component') {
+          await this.ctx.actions.wizard.detectFrameworks()
+          await this.ctx.actions.wizard.initialize()
+        }
```

Comment:
The guts of this function is now almost identical to the `setAndLoadCurrentTestingType` mutation.  Can these two areas be refactored to use the same code?  If it does not seem worth doing in this fix, then put some comments in both places to indicate that they should be updated together in the future.

Reply:
If this is the case, I think we should just do the refactor right here right now -- realistically, are we going to slate time to revisit a low value TODO in the future?